source/1_engine/rex_renderer_core/include/rex_renderer_core/graphics_api.h

```diff
@@ -0,0 +1,18 @@
+#pragma once
+
+namespace rex
+{
+    enum class GraphicsAPI
+    {
+        OpenGL,
+        DirectX
+    };
+
+    #ifdef REX_RENDERER_OPENGL
+        constexpr GraphicsAPI g_graphics_api = GraphicsAPI::OpenGL;
```

Comment:
makes this inline constexpr, you'd get linker errors or bigger obj files otherwise